### PR TITLE
Add New Seasons Market (US) and fix LinkedData parsing of OpeningHoursSpecification when only dealing with a dict

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Fix https://github.com/alltheplaces/alltheplaces/issues/7890

### CODE REVIEW

**Comment by CloCkWeRX** [comment]
Hmm, 8 results but 21 listed on the storefinder page.

File: locations/hours.py [hunk 1]
```diff
@@ -640,7 +640,6 @@ def add_range(self, day, open_time, close_time, time_format="%H:%M"):
 
         if day not in DAYS:
             raise ValueError(f"day must be one of {DAYS}, not {day!r}")
```
Comment (CloCkWeRX) [inline locations/hours.py:642]:
```suggestion
            raise ValueError(f"day must be one of {DAYS}, not {day!r}")

```

**Comment by CloCkWeRX** [comment]
@Cj-Malone I feel like you've done a bit on the linked data parser, so pinging you for review specifically.

Let me know if we'd like to split this into two PRs - fixing linked data parsing vs new spider.

**Comment by Cj-Malone** [comment]
Yeah I wrote from_linked_data before I wrote LinkedDataParser, I guess I'd like to move it in there, but it'll end up as a big refactor. I'll take a proper look at this tomorrow.

I'm not even sure a straight dict is valid in the spec. Yeah Google's validator seems to work, but that's just one implementation. With this, if it's fairly common, but against spec, I'd have no issue implementing it in shared code. If it's rare, I usually prefer it fixed in pre parse.

**Comment by CloCkWeRX** [comment]
https://www.w3.org/TR/json-ld11/#representing-singular-values-as-arrays I think is the bit of the JSON LD spec that is relevant.

> Generally, when compacting, properties having only one value are represented as strings or [maps](https://infra.spec.whatwg.org/#ordered-map), while properties having multiple values are represented as an array of strings or [maps](https://infra.spec.whatwg.org/#ordered-map). This means that applications accessing such properties need to be prepared to accept either representation.

**Review by Cj-Malone — APPROVED** [review]
Seems fine thanks. As I've added in the code, I'd like to deprecate oh.from_linked_data in favour of ld.parse_opening_hours, it's where the code belongs. I see no reason to rush this though.

**Pull request merged**